compute/cker/src/train/FullyConnected.test.cc

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
How about moving it in `test` directory?

`compute/cker/include/cker/train/test`